### ISSUE TITLE
build error for bastion

### ISSUE THREAD

**Issue by jtwalsh0** [dssg/dirtyduck#5]
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libpq-dev : Depends: libpq5 (= 10.4-2.pgdg80+1) but 9.6.7-0+deb9u1 is to be installed
 postgresql-9.6-pgrouting : Depends: postgresql-9.6 but it is not going to be installed
                            Depends: postgresql-9.6-postgis-2.4 but it is not going to be installed
                            Depends: libboost-atomic1.55.0 but it is not installable
                            Depends: libboost-chrono1.55.0 but it is not installable
                            Depends: libboost-date-time1.55.0 but it is not installable
                            Depends: libboost-system1.55.0 but it is not installable
                            Depends: libboost-thread1.55.0 but it is not installable
                            Depends: libcgal10 but it is not installable
 postgresql-contrib-9.6 : Depends: postgresql-9.6 (= 9.6.9-2.pgdg80+1) but it is not going to be installed
                          Depends: libssl1.0.0 (>= 1.0.0) but it is not installable
E: Unable to correct problems, you have held broken packages.
ERROR: Service 'bastion' failed to build: The command '/bin/sh -c sh -c "echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' > /etc/apt/sources.list.d/pgdg.list" &&      wget --quiet -O - http://apt.postgresql.org/pub/repos/apt/ACCC4CF8.asc | apt-key add - &&      apt-get -y update &&      apt-get -y install less postgresql-9.6-postgis-2.2      postgresql-contrib-9.6      libpq-dev postgresql-9.6-pgrouting' returned a non-zero code: 100
```

**Comment by jtwalsh0** [comment]
Ah, looks like the [v3.beta](https://github.com/dssg/dirtyduck/tree/v3.beta) branch is tackling this

**Comment by nanounanue** [comment]
Yup I need to merge that. I will do that today

**Comment by nanounanue** [comment]
Fixed in 5b19764760baa2a7cb81f8af23d875fedd776103
